tests/lib/rules/jsx-uses-vars.js

```diff
@@ -30,14 +30,14 @@ const parserOptions = {
 // -----------------------------------------------------------------------------
 
 const ruleTester = new RuleTester({ parserOptions });
-const linter = ruleTester.linter || eslint.linter || eslint.Linter;
-linter.defineRule('jsx-uses-vars', require('../../../lib/rules/jsx-uses-vars'));
+const ruleDefiner = ruleTester[Symbol.for('react.RuleTester.RuleDefiner')] || ruleTester.linter || eslint.linter || eslint.Linter;
```

Comment:
same here

Reply:
Done in https://github.com/jsx-eslint/eslint-plugin-react/pull/3759/commits/b0f5afd085e033c99a70d8bcd1dcb7e04fe7cda5.